src/lib/isLicensePlate.js

```diff
@@ -15,8 +15,23 @@ const validators = {
     /^[A-Z]{3}[ -]?[0-9][A-Z][0-9]{2}|[A-Z]{3}[ -]?[0-9]{4}$/.test(str),
   'sv-SE': str =>
     /^[A-HJ-PR-UW-Z]{3} ?[\d]{2}[A-HJ-PR-UW-Z1-9]$|(^[A-ZÅÄÖ ]{2,7}$)/.test(str.trim()),
+  'en-IN': str => /^[A-Z]{2}[ -]?[0-9]{1,2}(?:[ -]?[A-Z])(?:[ -]?[A-Z]*)?[ -]?[0-9]{4}$/.test(str),
 };
 
+validators['hi-IN'] = validators['en-IN'];

```

Comment:
nit: missed this review, perhaps this wasn't necessary. At some point, we actually contemplated to just use country-code only in the locales for stuff like license plates, that really don't depend on language per se.

@rubiin